src/devices/Vcnl4040/README.md

```diff
@@ -0,0 +1,111 @@
+# VCNL4040
+The VCNL4040 is a proximity and ambient light sensor device designed for various applications that require accurate measurement of proximity and light levels.
+
+## Key features of the sensor
```

Comment:
Markdown 101: all titles needs to be separated with a new line before and after (except the first one as no need before). You'll have to adjust almost all the page to follow the proper style.